.circleci/config.yml

```diff
@@ -40,9 +40,9 @@ templates:
     PARITY_URL_LINUX: 'https://releases.parity.io/ethereum/v2.3.3/x86_64-unknown-linux-gnu/parity'
     PARITY_URL_MACOS: 'https://releases.parity.io/ethereum/v2.3.3/x86_64-apple-darwin/parity'
     PARITY_VERSION: 'v2.3.3'
-    SOLC_URL_LINUX: 'https://github.com/ethereum/solidity/releases/download/v0.4.23/solc-static-linux'
+    SOLC_URL_LINUX: 'https://github.com/ethereum/solidity/releases/download/v0.5.4/solc-static-linux'
     SOLC_URL_MACOS: 'https://www.dropbox.com/s/4amq3on2ds1dq36/solc_0.4.23?dl=0'
```

Comment:
@palango or @ulope How was that dropbox link generated? Where did you find it? Can you find/make one for solc v0.5.4 ?

Reply:
@ulope clarified that we don't run any OSX tests

Ergo this can go away.